src/shared/components/post/post-listing.tsx

```diff
@@ -386,10 +386,10 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
             </span>
           )}
           {this.props.showCommunity && (
-            <span>
-              <span className="mx-1"> {i18n.t("to")} </span>
-              <CommunityLink community={post_view.community} />
-            </span>
+            <>
+              {" "}
```

Comment:
Is the space necessary?

Reply:
Yeah, unfortunately, otherwise these elements cram together:

<img width="214" alt="Screenshot 2023-06-18 at 11 21 26 PM" src="https://github.com/LemmyNet/lemmy-ui/assets/643417/deb35498-2ffb-4ead-b64a-2a5b13eb4a23">